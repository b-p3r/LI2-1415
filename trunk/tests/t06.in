5 3
ELP
OAI
SRE
ARC
AIO
