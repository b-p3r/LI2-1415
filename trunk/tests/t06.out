A
AI
AIA
AIAS
AIO
AIROSA
ALE
ALI
ALIE
ALO
AO
AOS
AR
ARA
ARAR
ARARA
ARARAS
ARAS
ARCO
ARE
AREA
AREAL
AREAS
AREIA
AREIAS
ARIA
ARIAS
ARO
AROS
ARRAIA
ARRASA
ARRASO
ARRE
ARREIA
ARROLA
AS
ASA
ASO
CE
CEAR
CEIA
CEIAS
CERA
CERAS
CERRA
CERRO
CIO
COR
CORA
CORAR
CORARIA
CORE
COREI
COREIA
CORRA
CORRAS
CORRE
CORREIA
CORREIAS
CORRI
CORRIA
CORRO
CORROA
CORSA
CORSO
CRE
CREIA
CRER
CRI
CRIA
CRIAR
CRIARA
CRIARAO
CRIAREI
CRIARIA
CRIAS
CRIE
CRIO
E
ECO
ECRA
ECRAS
EIA
EIRA
EIRAS
ELA
ELAS
ELO
ELOS
ERA
ERARIO
ERAS
ERRA
ERRO
ERROS
IA
IAO
IAS
ICE
IR
IRA
IRAO
IRAS
IRE
IREI
IRRA
IRREAL
LA
LAR
LAS
LE
LEAO
LEOA
LEOAS
LI
LIA
LIRA
LIRAS
LO
LOS
O
OCRE
OE
OLA
OLE
OLEAR
ORA
ORAI
ORAL
ORAR
ORAS
ORCE
ORE
OS
PA
PAI
PAIRA
PAIRAR
PAIRARA
PAIRE
PAO
PAR
PARA
PARAR
PARARA
PARAREI
PARARIA
PARAS
PARCERIA
PARCERIAS
PARCO
PARE
PARECIA
PARECIAS
PARECO
PAREI
PARI
PARO
PAROLE
PARRA
PAS
PI
PIA
PIAO
PIAR
PIAS
PIE
PILA
PILAO
PILAR
PILE
PIRA
PIRAR
PIRAS
PIROSA
RA
RAIA
RAIAR
RAIAS
RAIO
RALE
RALI
RALO
RALOS
RAP
RARA
RARAS
RAREIA
RARO
RAROS
RAS
RASA
RASAR
RASO
RE
REAL
RECO
RECRIA
REI
RI
RIA
RIAS
RICO
RIO
RIPA
RIPAS
RO
ROA
ROCE
ROI
ROIA
ROL
ROLA
ROLAS
ROLE
ROSA
ROSAL
ROSARIO
SA
SAI
SAIA
SAIO
SAIR
SAIRA
SAIREI
SAL
SAO
SARA
SARAR
SARE
SARRO
SO
SOA
SOAR
SOARA
SOE
SOL
SOLA
SOLAR
SOR
SORRI
SORRIA
SORRIO
